RecoLocalCalo/HGCalRecProducers/python/HGCalUncalibRecHit_cfi.py

```diff
@@ -45,9 +45,16 @@
         ),
 
     HGCHEBConfig = cms.PSet(
-        isSiFE  = cms.bool(False),
+        #isSiFE  = cms.bool(False), # GF
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant

Reply:
hello @slava77 , not needed, thanks for spotting.